src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -123,7 +124,7 @@
                 window-height (if selected-item (- height 72) height)
                 {:keys [keyboard-shown]} (hooks/use-keyboard)
                 bg-height-expanded (- window-height (:top insets))
-                bg-height (max (min @content-height bg-height-expanded) 200)
+                bg-height (max (min @content-height bg-height-expanded) 109)
```

Comment:
**PROBLEM:**

The drawer has a min height of `200`, which looks good if the bottom sheet has 3 or more `action-drawer` items. If we *reduce* the number of `action-drawer` items to 2 or below. It leaves a space at the bottom.

![imgonline-com-ua-twotoone-kbxQHR9h4j copy](https://user-images.githubusercontent.com/19339952/216661183-7412bd00-98a8-4ff6-a234-d29324993ab6.jpg)

**SOLUTION:**

I reduced the minimum height to `109` (as per Figma) to accommodate one action-drawer item and leave unwanted spaces at the bottom.

cc: @ilmotta - https://github.com/status-im/status-mobile/pull/14952#pullrequestreview-1279538720

Reply:
Excellent @smohamedjavid!